src/db/incidentStorage.js

```diff
@@ -23,10 +23,9 @@ class IncidentStorage {
   async import (content, sha, filePath) {
     await this.db.ready
     const newEntry = {}
-    newEntry.content = Buffer.from(content, 'base64').toString('utf-8')
+    newEntry.content = content
```

Comment:
Warum ist es nicht mehr benötigt den Inhalt der Dateien von base64 zu utf-8 umzuwandeln?
Wird das wo anders gemacht? Oder ist `content` schon in utf-8?

Reply:
Okay, das hat sich geklärt. Mit der Frontmatter Pull Request ergibt es sinn.